content/en/real_user_monitoring/browser/modifying_data_and_context.md

```diff
@@ -29,6 +29,74 @@ There are various ways you can modify the [data collected][1] by RUM, to support
 - Reducing how much RUM data you're collecting, through sampling the data.
 - Providing more context than what the default attributes provide about where the data is coming from.
 
+## Override default RUM view names
+
+The RUM SDK automatically generates a [view event][14] for each new page visited by your users, or when the page URL is changed (for single page applications). A view name is computed from the current page URL, with variable alphanumeric ids being removed automatically (for example, "/dashboard/1234" becomes "/dashboard/?").
+
+Starting with [version 2.17.0][15], you may specify your own view names by tracking view events manually with the `trackViewsManually` option:
+
+1. Set up `trackViewsManually` to true when initializing RUM.
+{{< tabs >}}
+{{% tab "NPM" %}}
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum';
+
+datadogRum.init({
+    ...,
+    trackViewsManually: true,
+    ...
+});
+```
+{{% /tab %}}
+{{% tab "CDN async" %}}
+```javascript
+DD_RUM.onReady(function() {
+    DD_RUM.init({
+        ...,
+        trackViewsManually: true,
+        ...
+    })
+})
+```
+{{% /tab %}}
+{{% tab "CDN sync" %}}
+```javascript
+window.DD_RUM &&
+    window.DD_RUM.init({
+        ...,
+        trackViewsManually: true,
+        ...
+    });
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+2. Start views for each new page, or route change (for single page applications), and define the associated view name. If not provided a name, the view name defaults to the page URL path.
```

Comment:
You **must** start views for each new page, or route change (for single page applications), and optionally define the associated view name (it defaults to the page URL path). No RUM data gets collected until the view is started.